frontend/view-edit-client/interactions/single-interaction-slat.component.tsx

```diff
@@ -16,6 +16,10 @@ import {
 export default React.forwardRef<any, SingleClientInteractionProps>(
   function SingleClientInteraction(props, ref) {
     const scope = useCss(css);
+    const [interactionKind, setInteractionKind] = React.useState<
+      InteractionKind
+    >(InteractionKind.cuService);
+    const followUpsEnabled = localStorage.getItem("follow-ups") === "true";
```

Comment:
This is a feature toggle that lets us work on the follow ups UI without showing it to CU staff yet